server.py

```diff
@@ -61,46 +92,49 @@ def internal_server_error(error):
 ######################################################################
 # API placeholder
 ######################################################################
-@app.route('/')
+@app.route(PATH_ROOT)
 def index():
     """ Return something useful by default """
-    return jsonify(name='Inventory Root'), HTTP_200_OK
+    return jsonify(name='Inventory Root'), status.HTTP_200_OK
 
-@app.route('/inventory', methods=['GET'])
+@app.route(PATH_INVENTORY, methods=[GET])
 def list_inventory():
     """ Return all entries in the Inventory system """
     results = [prod_info.serialize() for prod_info in ProductInformation.list_all()]
-    return jsonify(results), HTTP_200_OK
+    return jsonify(results), status.HTTP_200_OK
 
-@app.route('/inventory/<int:prod_id>', methods=['GET'])
+@app.route(PATH_INVENTORY_PROD_ID, methods=[GET])
 def get_prod_info(prod_id):
     """ Return ProductInformation identified by prod_id """
     prod_info = ProductInformation.find(prod_id)
     if not prod_info:
         raise NotFound("Product with id '{}' was not found in Inventory".format(prod_id))
-    return jsonify(prod_info.serialize()), HTTP_200_OK
+    return jsonify(prod_info.serialize()), status.HTTP_200_OK
 
-@app.route('/inventory', methods=['POST'])
+@app.route(PATH_INVENTORY, methods=[POST])
 def create_prod_info():
     """
     Creates a ProductInformation
     This endpoint will create a ProductInformation based the data in the body that is posted
     """
-    check_content_type('application/json')
+    check_content_type(JSON)
     prod_info = ProductInformation()
     prod_info.deserialize(request.get_json())
+
+    if (prod_info.find(prod_info.prod_id)):
```

Comment:
find() is a static method, so maybe use ProductionInformation.find() instead to avoid future confusion.

Reply:
Addressed